StringCalculator.java

```diff
@@ -0,0 +1,47 @@
+public class StringCalculator {
+
+    public static void main(String ... args) {
+        StringCalculator stringCalculator = new StringCalculator();
+        System.out.println(stringCalculator.result("1 + 2"));
+    }
+
+
+    public int result(String input) {
+        String[] inputs = input.split(" ");
+        return calculator(inputs.length -1, inputs);
```

Comment:
```suggestion
        return calculator(inputs.length - 1, inputs);
```
공백을 추가해주면 좋을 것 같네요😎 

그리고 전체적으로 매직 넘버를 상수화해주면 어떨까요? 상수화한다면 어떤 장점을 가질까요?